helpers/yojson_ext.rei

```diff
@@ -0,0 +1,3 @@
+let with_yojson_string:
```

Comment:
This appears to be dead code: it's used in one place, but that place appears to be dead code.

Non blocking: This function is also very confusing - it's hard to figure out what it does and should have some docstrings.

cc @EduardoRFS

Reply:
@d4hines please open an issue or a PR removing this.